spec/models/ems_folder_spec.rb

```diff
@@ -3,6 +3,35 @@
 
   include_examples "MiqPolicyMixin"
 
+  describe "AggregationMixin methods" do
+    let(:host) { FactoryBot.create(:host) }
+    let(:rp) { FactoryBot.create(:resource_pool) }
+    let(:vm) { FactoryBot.create(:vm_vmware, :hardware => FactoryBot.create(:hardware, :cpu2x2, :memory_mb => 2048)) }
+    let(:vm2) { FactoryBot.create(:vm_vmware, :hardware => FactoryBot.create(:hardware, :cpu2x2, :memory_mb => 1024)) }
+
+    before do
+      subject.with_relationship_type("ems_metadata") { subject.set_parent host }
```

Comment:
are there non `relationship_types` ways to do this?
Something like a `subject.update_attributes(:bio_parent => host)`

Reply:
answer: resource pools and folders don't have these fluffy methods
and since we want to get rid of them, lets just get tests in there and worry about it later

Reply:
yeah, so the reason this won't work, for anyone looking in the future, is that folders use the relationship route on https://github.com/ManageIQ/manageiq/blob/846cc9fc0fec428f318644f6ec531bc2308ab256/app/models/ems_folder.rb#L66 for the aggregation methods these tests are running.